react/ProductCustomizer.js

```diff
@@ -1,54 +1,180 @@
-  handleCloseModal() {
+  /**
+  * handleCloseModal
+  * Close the product modal
+  * @return void
+  */
+  handleCloseModal = () => {
     this.setState({ isModalOpen: false })
   }
 
+  handleSubmit = e => {
+    console.log(this.state.selectedVariations)
+    e.preventDefault()
+
+    // Checl Lists validations
+    // Add product to cart
+  }
+
   render() {
+    const {
+      variations,
+      selectionsValidated,
+      onHandleUpdateAmount,
+    } = this.state
+
+    const { enableChangeToppings } = this.props
+
     return (
       <div>
         <Button onClick={this.handleOpenModal}>Customize your product</Button>
 
         <Modal
-          top
-          isMobile="true"
+          centered
           isOpen={this.state.isModalOpen}
           onClose={this.handleCloseModal}
         >
           <div className={'flex-ns h-100-ns'}>
-            <div className={'w-100 w-third-ns bg-light-gray bb b--light-gray flex-ns items-center-ns  h-100-ns'}>
-              <h1 className={`vtex-product-customizer__title tc f4 fw5 absolute ma0 pa5 w-100 bg-black-40 white ${!isMobile ? 'dn' : ''}`}>Create Your Own</h1>
+            <h1 className={`vtex-product-customizer__title tc f4 fw5 ma0 pa5 w-100 bg-black-40 white ${!isMobile ? 'dn' : ''}`}>Create Your Own</h1>
+            <div className={'w-100 w-third-ns flex-ns tc items-center-ns pa5 h-100-ns'}>
               <img
-                className={'vtex-product-customizer__image'}
-                alt="Create Your Own"
-                src="https://via.placeholder.com/670x500"
+                className={'vtex-product-customizer__image br3'}
+                alt="Product Customize Image"
+                src="https://via.placeholder.com/330x330"
               />
             </div>
             <div className={'w-100 w-two-thirds-ns flex-ns flex-column-ns'}>
-              <h1 className={`vtex-product-customizer__title fw5 ma0 f3 pa5 ${isMobile ? 'dn' : ''}`}>Create Your Own</h1>
-              <div className={'pb5-ns pt0-ns ph5-ns pa5 bb b--light-gray'}>
-                <p className={'ma0 fw3'}>Go freestyle! Simply choose your sauce and add any toppings.</p>
-              </div>
-              <AttachmentsList />
+              <form name="vtex-product-customizer-form" onSubmit={this.handleSubmit}>
+                <h1 className={`vtex-product-customizer__title fw5 ma0 f3 pa5 ${isMobile ? 'dn' : ''}`}>Create Your Own</h1>
+                <div className={'pb5-ns pt0-ns ph5-ns  ph5 pb5 bb b--light-gray'}>
+                  <p className={'ma0 fw3'}>Todo el sabor mexicano. Chorizo, pico de gallo, jalapeño y tocino.</p>
+                </div>
+                <div className={'vtex-product-customizer__options bg-light-gray bg-transparent-ns overflow-auto'}>
+                  <h4 className={'ma0 pv3 ph5'}>
+                    <span className={'f5 fw5'}>Select item variation</span>
+                  </h4>
+                  {variations.map((variation, key) => {
+                    return (
```

Comment:
There is no need for this `return` statement, you can use `variations.map((variation, key) => ( the content inside the return statement ))`

Reply:
Fixed in commit 2d2092c522f572a871eea9c57d10a73b4837ee5f